ci/checks/style.sh

```diff
@@ -20,6 +20,10 @@ PATH=/conda/bin:$PATH
 # Activate common conda env
 source activate gdf
 
+gpuci_conda_retry remove --force rapids-build-env rapids-notebook-env
+gpuci_conda_retry install -y "clang=11.0.0"
+gpuci_conda_retry install -y "clang-tools=11.0.0"
```

Comment:
These lines can be removed. Once the version is updated in integration, our images will come with the correct package.

Reply:
Done!